### PR TITLE
Profile Page - Display Date Joined

### LINKED ISSUE #1078: Update OC Web Based Learning Scholarship Policy
As a user, after I login at `/login`, I will be redirected to `/profile` page. If I checked the "Want To Receive Scholarship Info" box during registration, I should see a LinkButton which opens [this link](https://drive.google.com/file/d/1s4PsrUzs2MU4itCA44x1VBzX7Gd77H8n/view?usp=sharing) in a new tab.

Notes for devs:
- API Documentation is here: https://api.operationcode.org/docs/
- You'll want to GET `/auth/user` for the boolean `wantsScholarshipInfo` on the `/profile` page to conditionally render the link
- The link should use the [LinkButton](https://operation-code-storybook.netlify.app/?path=/story/linkbutton--default) component like so:
```jsx
<LinkButton
  href="https://drive.google.com/file/d/1s4PsrUzs2MU4itCA4…"
  analyticsEventLabel="Viewed Scholarship Policy"
  fullWidth
  theme="secondary"
>
  View Scholarship Policy
</LinkButton>
```
- The link should be the last of the 3 buttons on the profile page.

### LINKED ISSUE #1222: Profile Page - Display Date Joined
# Bug Report
## What is the current behavior?
On Profile Page Date Joined currently displays: "lid Date"

## What is the expected behavior?
Display Date Joined in MM-DD-YYYY format

## What steps did you take to get this behavior?
Used **date-fns** from https://date-fns.org/ to display date

### PR DESCRIPTION
**Pull request by calincionca35**
# Description of changes
Profile Page - Display Date Joined

# Issue Resolved
Resolves #1078 
Resolves #1222

### CODE REVIEW

**Comment by kylemh** [comment]
#### 📝Changed routes:


| [**/events**](https://operation-code-git-master.operation-code.now.sh/events) | [**/profile**](https://operation-code-git-master.operation-code.now.sh/profile) |
|:-:|:-:|
|<a href="https://operation-code-git-master.operation-code.now.sh/events">Screenshot of /events</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/cw2XqDYW7bc3loKq85lVbWJFG60_.png">(view full size)</a> |<a href="https://operation-code-git-master.operation-code.now.sh/profile">Screenshot of /profile</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/EgHE_tsj3ej_mL238j_mPafCBAQ_.png">(view full size)</a> |



| [**/who_we_serve**](https://operation-code-git-master.operation-code.now.sh/who_we_serve) |
|:-:|
|<a href="https://operation-code-git-master.operation-code.now.sh/who_we_serve">Screenshot of /who_we_serve</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/dVIMNHIYPJe4MS2AmeQX21CyHb8_.png">(view full size)</a> |







Commit 076bec666f1dbbdea18644029740fa43d3307557 (https://operation-code-git-master.operation-code.now.sh).

**Review by kylemh — APPROVED** [review]
Looks great to me!

#1078 asks that you make a GET request for user info on their desire to hear about scholarships to conditionally render the informational button, but the link itself renders a document which helps people find out if they qualify or not. Additionally, we havent had an input in the form _asking_ if people are interested, so that data seems unreliable.

tl;dr - this is perfect
